transitmatters/new-train-tracker
# feat: show googly eyes 👀

## Motivation


👀show the trains with googly eyes

https://apnews.com/article/boston-subway-trains-googly-eyes-42c69329096151eba01dc5b31f34fedc 

## Changes


![image](https://github.com/transitmatters/new-train-tracker/assets/46619169/5f0659fe-4a17-4d59-9c36-17263ed66d98)

## Testing Instructions
Look at trains and verify that the cars with confirmed googly eyes show up with the eyes emoji. Let me know if there are any other cars I can add!
